react-components/src/components/Toolbar/Image360HistoricalSummary.tsx

```diff
@@ -0,0 +1,140 @@
+/*!
+ * Copyright 2023 Cognite AS
+ */
+
+import { Detail, Flex } from '@cognite/cogs.js';
+import React from 'react';
+import styled from 'styled-components';
+import { Thumbnail } from '../utils/Thumbnail';
+import { Cognite3DViewer, Image360 } from '@cognite/reveal';
+import { uniqueId } from 'lodash';
+
+export interface Image360RevisionDetails{
+  date?: string;
+  imageUrl?: string;
+  index?: number;
+  image360Entity?: Image360;
+};
+
+export interface Image360HistoricalSummaryProps{
+  stationId?: string;
+  stationName?: string;
+  revisionCollection?: Image360RevisionDetails[];
```

Comment:
Same as with `Image360HistoricalDetails` - do we need this many arguments, can we do with just the 360 image entity?